core/src/test/kotlin/ch/ergon/dope/resolvable/clause/OrderByClauseTest.kt

```diff
@@ -91,7 +148,8 @@ class OrderByClauseTest : ManagerDependentTest {
             queryString = "SELECT $1 ORDER BY `stringField` ASC",
             DopeParameters(positionalParameters = listOf(parameterValue)),
         )
-        val underTest = SelectOrderByTypeClause(someStringField(), OrderByType.ASC, someSelectClause(parameterValue.asParameter()))
+        val underTest =
+            SelectOrderByClause(OrderExpression(someStringField(), ASC), parentClause = someSelectClause(parameterValue.asParameter()))
```

Comment:
can you break up this line? like on line 179

Reply:
done